cmake/VASTPackage.cmake

```diff
@@ -47,4 +45,45 @@ set(CPACK_DEBIAN_PACKAGE_CONTROL_EXTRA
     "${CMAKE_CURRENT_SOURCE_DIR}/scripts/debian/prerm")
 set(CPACK_DEBIAN_PACKAGE_DEBUG ON)
 
+# For the static binary builds it doesn't make much sense to install development
+# utilities, so we never do so.
+if (VAST_ENABLE_STATIC_EXECUTABLE)
+  get_cmake_property(CPACK_COMPONENTS_ALL COMPONENTS)
+  list(REMOVE_ITEM CPACK_COMPONENTS_ALL "Development" "Unspecified")
+endif ()
```

Comment:
This is good idea, although for some strange reason this doesn't seem to work with the `DEB` generator.

Reply:
We hope that this is fixed by aa3b456d894a68c1c4aa35c6251507f8d9d5edab.